### PR TITLE
 Client resubmit the Affidavit Info or update Account Info#7781

### PR DESCRIPTION
**Pull request by shabeeb-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/7781

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/create-account/AccountCreateBasic.vue [hunk 1]
```diff
@@ -195,7 +197,10 @@ export default class AccountCreateBasic extends Mixins(Steppable) {
     if (this.isFormValid()) {
       // if its not account change , do check for duplicate
       // if its account change , check if user changed the already existing name
-      const checkNameAVailability = !this.isAccountChange || (this.orgName !== this.currentOrganization?.name)
+      // const checkNameAVailability = !this.isAccountChange || (this.orgName !== this.currentOrganization?.name)
```
Comment (sumesh-aot) [inline auth-web/src/components/auth/create-account/AccountCreateBasic.vue:200]: remove this if not needed?

File: auth-web/src/components/auth/create-account/AccountTypeSelector.vue [hunk 2]
```diff
@@ -235,10 +238,11 @@ export default class AccountTypeSelector extends Mixins(Steppable) {
   }
 
   private selectAccountType (accountType) {
+    // removed below code for becid re-upload. need to persisit all data in current org
     // to reset any existing details ;user might have went to user profile ;came back and selects another type scenarios
-    if (!this.isAccountChange) {
-      this.resetCurrentOrganisation()
-    }
+    // if (!this.isAccountChange) {
```
Comment (sumesh-aot) [inline auth-web/src/components/auth/create-account/AccountTypeSelector.vue:243]: remove unused code

File: auth-web/src/components/auth/create-account/PaymentMethodSelector.vue [hunk 3]
```diff
@@ -32,10 +35,11 @@
           data-test="save-button"
           :disabled="!isEnableCreateBtn"
         >
-          Create Account
+         {{ readOnly ? 'Submit' : 'Create Account'}}
```
Comment (sumesh-aot) [inline auth-web/src/components/auth/create-account/PaymentMethodSelector.vue:38]: hmm this variable name is confusing. Submit if it's read-only? May be rename it for better readability.

**Review by sumesh-aot — COMMENTED** [review]
Add some test cases too

File: auth-web/src/components/auth/create-account/PaymentMethodSelector.vue (hunk 3 above)
Comment (shabeeb-aot) [reply auth-web/src/components/auth/create-account/PaymentMethodSelector.vue:39]: added more comments

**Review by karthik-aot — APPROVED** [review]

**Review by saravanpa-aot — APPROVED** [review]

**Pull request merged**